pkgs/by-name/zo/zotify/package.nix

```
@@ -0,0 +1,58 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "zotify";
```

```suggestion
{
  fetchFromGitHub,
  python3,
  lib,
}:

python3.pkgs.buildPythonApplication rec {
  pname = "zotify";
```

Will make the diff in the future significantly smaller if an override is needed.

---

I don't quite understand this comment, can you please explain how that's going to make the diff smaller?

---

> ...how that's going to make the diff smaller?

When an override is needed. `protobuf` is already relaxed, thus there is a chance that we need an override for this sooner than later.

---

why would we need an override though?